distribution/examples/security/api-key/rbac/README.md

```
@@ -7,58 +7,76 @@ Secure endpoints using API keys combined with role-based access control (RBAC).
 
-```xml
-<spring:bean class="com.predic8.membrane.core.interceptor.apikey.stores.ApiKeyFileStore">
-    <spring:property name="location" value="./demo-keys.txt" />
-</spring:bean>
+```yaml
+components:
+   keys:
+     apiKeyFileStore:
+       location: ./demo-keys.txt
 ```
 **OR**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)